jenkins/runlocal

```diff
@@ -12,19 +15,13 @@ export PB_SERVER_IMAGE_TAG=${USER}
 # is used by the pod.
 export PB_SERVER_IMAGE_PULL_POLICY=never
 
-# We explicitly set WORKSPACE_TMP so that RPM builds (which build images using
-# jenkins/run) will store the resulting RPM in that work space temporary
-# directory.
-export WORKSPACE_TMP=/var/tmp/${USER}/runlocal
-mkdir -p /var/tmp/${USER}
-rm -rf ${WORKSPACE_TMP}
-mkdir ${WORKSPACE_TMP}
+# Create an RPM from the current source tree.
+make -C server/rpm clean rpm
 
-# Create a RHEL 9 RPM from the current source tree.
-make -C server/rpm clean rhel-9-rpm
+source /etc/os-release
 
 # Build the canned Pbench Server container from the RPM built above.
-RPM_PATH=${WORKSPACE_TMP}/rpmbuild/RPMS/noarch/pbench-server-*.rpm bash -ex ./server/pbenchinacan/container-build.sh
+BASE_IMAGE=${ID}:${VERSION_ID} RPM_PATH=${HOME}/rpmbuild/RPMS/noarch/pbench-server-*.rpm server/pbenchinacan/container-build.sh
```

Comment:
```
$ jenkins/runlocal
make: Entering directory '/home/wscales/pbench/server/rpm'
Building clean rpm for pbench-server-1.0.0-9g05af2ec06 from /home/wscales/rpmbuild/TMP to /home/wscales/rpmbuild

...

+ exit 0
make: Leaving directory '/home/wscales/pbench/server/rpm'
Resolved "centos" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
Trying to pull quay.io/centos/centos:9...
Error: initializing source docker://quay.io/centos/centos:9: reading manifest 9 in quay.io/centos/centos: manifest unknown: manifest unknown
```
I figured that approach was too good to be true.  😢

Reply:
CentOS-9 is labeled `stream9`.

Reply:
What will work for CentOS?

Reply:
Latest update should address that.

Reply:
Further updates now only support CentOS Stream 9 and RHEL 9.